test/DivConformingTests.jl

```diff
@@ -114,7 +114,7 @@ function main(parts)
       hid_to_gid,
       hid_to_part)
 
-    model = GridapDistributed.DistributedDiscreteModel(models,gids)
+    model = GridapDistributed.GenericDistributedDiscreteModel(models,gids)
```

Comment:
why it is needed to call here to the `GenericDistributedDiscreteModel` constructor?

Reply:
I guess it's not, but calling `DistributedDiscreteModel` would just default to this constructor. I suppose it would get inlined though. I can change it if you want, I just thought it would be more explicit.

Reply:
Solved in [939bf22](https://github.com/gridap/GridapDistributed.jl/pull/98/commits/939bf22000d73aad02f75fef23bec37ef70bbac4).

Reply:
Do we need a fully qualified name after the change? i.e., to prefix with  `GridapDistributed.`